buildlib/azure-pipelines-release.yml

```diff
@@ -19,13 +19,26 @@ stages:
         displayName: build tarball and source rpm
         container: fedora
         steps:
-          - bash: ./autogen.sh
-            displayName: Setup autotools
+          - bash: |
+              set -eE
+              TAG=`git describe --tags`
+              success=$?
+              if [[ $success -eq 0 ]]
+              then
+                make -C bindings/java/src/main/native/ set-version JUCX_VERSION=${TAG:1}
+              else
+                echo "##vso[task.logissue type=error]No git tag to build release."
+                echo "##vso[task.complete result=Failed;]"
```

Comment:
I need `publish` to fail, because it may publish release not from a tag

Reply:
You need **publishing to Maven** fail, not the whole pipeline, which is also publishing a tarball & `src.rpm` to Github. Anyway, I don't see how this could happen, this pipeline is triggered on a tag creation, how it possibly can be missed? If we really need to double-check it, let's make this check the first one.

Reply:
I moved published to the last step, so if it'll fail only publish will fail.